### PR TITLE
[13.0][MIG] sale_invoice_line_note

### PR DESCRIPTION
**Pull request by kranokporn**
@kittiu Can you review this?
This module works by adding notes from sale order to invoices.

### CODE REVIEW

**Comment by kittiu** [comment]
Please change the subject to "[13.0][MIG] sale_invoice_line_note". The correct technical name of the module helps other search for it easily.

**Comment by kittiu** [comment]
You can remove file README.rst.
You only need to make sure the content in readme folder is correct. The README.rst will be auto created for you by ocabot.

**Comment by kittiu** [comment]
The travis error because of this, https://github.com/OCA/account-invoicing/pull/754
And not it is merged.

@kranokporn please rebase yours. Thank you.

**Comment by kranokporn** [comment]
OK, I already rebase. 
Thank you.

@kittiu

File: sale_invoice_line_note/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Akim Juillerat <akim.juillerat@camptocamp.com>
+
+ * Kranokporn Th. <kranokpornt@ecosoft.co.th>
```
Comment (kittiu) [inline sale_invoice_line_note/readme/CONTRIBUTORS.rst:3]:
```suggestion
* Kranokporn Th. <kranokpornt@ecosoft.co.th>
```

**Review by kittiu — APPROVED** [review]

**Review by anasuarez1 — APPROVED** [review]
Functional review ok!

Since everything looks right can we get this merged?

**Review by manuelcalerosolis — APPROVED** [review]
I have checked this PR and it works correctly, please merge.

**Comment by rafaelbn** [comment]
Hello @kranokporn ! could you try to fix?

**Comment by dreispt** [comment]
Superseded by #1624

**Pull request closed without merging**